client/packages/coldchain/src/Equipment/Components/PropertyInput.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  Autocomplete,
+  BasicTextInput,
+  Checkbox,
+  NumericTextInput,
+} from '@common/components';
+import { PropertyNodeValueType } from '@common/types';
+import React, { FC } from 'react';
+
+export type PropertyInput = {
+  valueType: PropertyNodeValueType;
+  allowedValues?: string[];
+  value: string | number | boolean | undefined | null;
+  onChange: (value: string | number | boolean | undefined) => void;
+};
+
+export const PropertyInput: FC<PropertyInput> = ({
+  valueType,
+  allowedValues,
+  value,
+  onChange,
+}) => {
+  switch (valueType) {
+    case PropertyNodeValueType.Boolean:
+      return (
+        <Checkbox
+          checked={value as boolean}
+          onChange={e => onChange(e.target.checked)}
+        />
+      );
+    case PropertyNodeValueType.Integer:
+    case PropertyNodeValueType.Float:
+      const valueInt = value as number;
+      return (
+        <NumericTextInput
```

Comment:
Numeric inputs are looking a bit ugly as the seem to line up at an arburary point.
Any thoughts on how to improve the display?